apps/native/app/src/stores/organizations-store.ts

```diff
@@ -72,6 +73,10 @@ export const organizationsStore = create<OrganizationsStore>(
         const uri = `${url}?w=${size}&h=${size}&fit=pad&fm=png`
         return { uri }
       },
+      getOrganizationNameBySlug(slug: string) {
+        const org = get().organizations.find((o) => o.slug === slug)
+        return org?.title ?? ''
```

Comment:
Some services have not set the organisations property when service errors. We should default to some message instead of empty string

Reply:
If I don't get a name I default to showing an icon and no title 😊 I think that is what we want to do
![Screenshot 2024-11-15 at 10 51 10](https://github.com/user-attachments/assets/b39cc76f-8719-448a-b540-b24623b9e6b2)